### ISSUE TITLE
FxA Profile updates

### ISSUE THREAD

**Issue by flekkowich** [mozilla/kitsune#3555]
Profile updates w/ the FxA integration

see wireframes in invision for interaction (not final designs)

Final Designs [Zeplin Link](https://zpl.io/bWNjwGp)

cc @akatsoulas @kkellydesign @mcranston18

**Comment by mcranston18** [comment]
@akatsoulas Just a reminder that we needs URLs for:

The need help URL:
![Screen Shot 2019-05-07 at 9 38 40 AM](https://user-images.githubusercontent.com/1489696/57303721-e9752d80-70ab-11e9-9238-6c471d7bd3f2.png)

Manage Your Firefox Account:

![Screen Shot 2019-05-07 at 9 39 06 AM](https://user-images.githubusercontent.com/1489696/57303747-f8f47680-70ab-11e9-9364-3c2a48b32c25.png)

**Comment by rleitan** [comment]
The links for: Manage Firefox Account button and for Need help are not there.
Edit settings and Manage watch list sections should not be displayed.
Also the View my profile and Edit my profile pages are not matching the designs.
@mcranston18 @kkellydesign

**Comment by mcranston18** [comment]
Will be fixed once https://github.com/mozilla/kitsune/pull/3636 is merged in

**Comment by mcranston18** [comment]
@rleitan These should all be fixed once #3636 is merged in

- I have added a link for `Need Help`.
- For `Manage Your Firefox Account`, I am linking to https://accounts.firefox.com/settings . @akatsoulas  Please let me know if this is not the desired behaviour
- I have confirmed with Rina that "Edit Settings" and "Manage watch list" should be visible.
- The profile view and edit pages should match designs

**Comment by rleitan** [comment]
> * The profile view and edit pages should match designs
View My profile issues:
- "My profile" instead of View My Profile
- Contributions and About username sections are not displayed ( not sure if the Contributions section should be displayed only after I start contributing?)

Edit My Profile issues:
-Choose Photo and Delete buttons should be  positioned as per designs: https://app.zeplin.io/project/5c940ea1d154600581d4377b/screen/5cc064149eb7ef265a226d71
-Display Name field should be marked as mandatory
-Timezone field should be displayed between City and Preferred language
-Make My email public should not be displayed
-Changing Password option should not be displayed
-Email should not be displayed

![2019-05-20_12h36_12](https://user-images.githubusercontent.com/41474886/58011900-eb071280-7afb-11e9-8d28-02d956a17327.png)


@mcranston18

**Comment by akatsoulas** [comment]
`Password change` has been addressed in another PR. @kkellydesign can you take a look for the other comments?

**Comment by akatsoulas** [comment]
The Display name is an optional field

**Comment by kkellydesign** [comment]
> "My profile" instead of View My Profile
fixed.

> Contributions and About username sections are not displayed ( not sure if the Contributions section should be displayed only after I start contributing?)

Both of these areas are wrapped in conditionals, so they should appear if the conditionals are correct. If you'd like me to test this, please let me know how to add a contribution to my local instance. 

> Choose Photo and Delete buttons should be positioned as per designs
fixed. There's no longer a delete button, but I believe that was discussed and removed elsewhere. 

> Timezone field should be displayed between City and Preferred language
fixed. 

> Make My email public should not be displayed
fixed. 

> Changing Password option should not be displayed. Email should not be displayed
These are wrapped in a conditional, so they will only be displayed to users who have not yet migrated to FF accounts.
